README.md

```diff
@@ -1,91 +1,129 @@
+---
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
 # wtf-workshop
 
 <!-- badges: start -->
+
 <!-- badges: end -->
 
+# 2022 RStudio Conference Overivew
+
+| Content                   | Slides                                                                                                                | Exercises                                                                                                                                                                                                                   |
```

Comment:
Maybe note somewhere that PDFs (or whatever the relevant rendered product is) are often available in workshop repos, but this table is meant to track the underlying source for someone who wants to revise, not just read.

Note that the Google Drive folder isn't world-readable. Folks who need access should ask Jenny.